### ISSUE TITLE
About IE Compatibility - Simple Fix

### ISSUE THREAD

**Issue by mrtgzl** [malsup/corner#48]
People who had problems with ie compatibility. Code below deactivate the script only for ie browser.

Fırst it checks the browser whether if ie or not. if your browser is ie i just put a comment like Do Nothing, if not just write your plugin codes.

// PUT THIS CODE BETWEEN JAVASCRIPT TAGS

var isIE/_@cc_on=1@_/;
if(isIE){
   //Do Nothing !
}else
{
    // CORNER PLUGIN CODES
    $("#mydiv").corner("bottom","3px");
}

**Comment by jakemac53** [comment]
The whole point of this is to make the rounded corners work in all browsers.....so I would argue this is not a good fix if you are having IE issues

**Comment by mrtgzl** [comment]
Yes, It's just a temparory solution for not receiving any javascript errors.

**Comment by jakemac53** [comment]
ok, but how about using jQuery's not() function instead (http://api.jquery.com/not/). I have not tested what is below but I believe something like this should work, and only not round the corners on input elements and textareas in IE.

// PUT THIS CODE BETWEEN JAVASCRIPT TAGS

var isIE/@cc_on=1@/;
if(isIE){
//Do not select input elements or textareas!
$("#mydiv").not("input, textarea").corner("bottom","3px");
}else
{
// CORNER PLUGIN CODES
$("#mydiv").corner("bottom","3px");
}
